docs/how-to-guides/creating-grc20.md

```diff
@@ -0,0 +1,166 @@
+---
+id: creating-grc20
+---
+
+# How to create a GRC20 Token
+
+## Overview
+
+This guide shows you how to write a simple _GRC20_ Smart Contract, or rather a [Realm](../explanation/realms.md), in [Gnolang (Gno)](../explanation/gno-language.md). For actually deploying the Realm, please see the [deployment](deploy.md) guide.
+
+Our _GRC20_ Realm will have the following functionality:
+
+- Minting a configurable amount of token.
+- Keeping track of total token supply.
+- Fetching the balance of an account.
+
+## Prerequisites
+
+We will proceed using the typical directory structure for a Realm found within the [simple-contract guide](simple-contract.md). It is also worthwhile to consult the [GRC20 interface](../../examples/gno.land/p/demo/grc/grc20/igrc20.gno) which we will be importing and utilizing within this guide.
+
+## 1. Importing token package
+For this realm, we'll want to import the `grc20` package as this will include the main functionality of our token factory realm.
+
+mytoken.gno
+```go
+package mytoken
+
+import (
+	"std"
+	"strings"
+
+	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/demo/users"
+)
+
+var (
+	mytoken   *grc20.AdminToken
+	admin std.Address = "g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj" // set admin account
+)
+
+// init is a constructor function that runs only once (at time of deployment)
+func init() {
+    // provision the token's name, symbol and number of decimals
+	mytoken = grc20.NewAdminToken("Mytoken", "MTKN", 4)
+
+    // set the total supply
+	mytoken.Mint(admin, 1000000*10000) // @administrator (supply = 1 million)
+}
+```
+
+In this code preview, we have:
+- Defined a new local variable `mytoken` and assigned that the type of pointer to `grc20.AdminToken`.
+- Defined and set the value of local variable `admin` to point to a specific gno.land address of type `std.Address`.
+- Set the value of `mytoken` (type `*AdminToken`) to equal the result of creating a new token and configuring its name, symbol + decimal representation.
+- Minted 1 million `Mytoken` and set the administrator as the owner of these tokens.
+
+## 2. Adding token functionality
+
+The following section will be about introducing Public functions to expose functionality imported from the [grc20 package](../../examples/gno.land/p/demo/grc/grc20).
+
+```go
+func TotalSupply() uint64 {
```

Comment:
We should automate these documents. Manual checking for updates will be difficult and time-consuming if we don't start automating them now.

I had positive experiences with this tool: [embedmd](https://github.com/campoy/embedmd) (made by a former colleague of @ajnavarro). We should consider using this tool or a similar one.

Reply:
It's a simple enough tool, although the regex matching is clunky.

It's a good start though, I've incorporated it into our workflow:
[d6ee4da](https://github.com/gnolang/gno/pull/1046/commits/d6ee4dacb16414964f4c499e341eb6be6d5a0c37)